ts/src/sourcemapper/sourcemapper.ts

```diff
@@ -249,8 +247,57 @@ async function createFromMapFiles(mapFiles: string[]): Promise<SourceMapper> {
   return mapper;
 }
 
+function isErrnoException(e: unknown): e is NodeJS.ErrnoException {
+  if (e instanceof Error && 'code' in e) return true;
+  else return false;
```

Comment:
This could be simplified to:

```suggestion
  return e instanceof Error && 'code' in e;
```